F/Flux/Compat.toml

```diff
@@ -38,7 +38,7 @@ ZipFile = "0.7-0.8"
 
 ["0.10.1-0"]
 AbstractTrees = "0.2-0.3"
-CuArrays = "1.6.0-1"
+CuArrays = "1.4.3-1."
```

Comment:
What does the `1.` mean here, as opposed to `1`?

Reply:
It was a typo; should be fixed now

Thanks!

Reply:
Ah, great.